### ISSUE TITLE
Accessibility - article graphs

### ISSUE THREAD

**Issue by tbl73** [benwbrum/fromthepage#801]
In the graph of linked articles in Subject-Overview, we have an accessibility issue: "image map area needs alt text.  The clickable links to the other articles are the map areas which should have alt-text.  I couldn't figure out how to add the alt-text, though - I assume it belongs in the graph.dot file, but I couldn't work out how to correctly add it.

**Comment by benwbrum** [comment]
This has already been handled by [an alt text tag](https://github.com/benwbrum/fromthepage/blob/development/app/views/article/show.html.slim#L20).
